barretenberg/cpp/src/barretenberg/dsl/acir_format/acir_format.hpp

```diff
@@ -173,9 +173,9 @@ struct AcirProgramStack {
     std::vector<AcirFormat> constraint_systems;
     WitnessVectorStack witness_stack;
 
-    AcirProgramStack(const std::vector<AcirFormat>& constraint_systems_in, const WitnessVectorStack& witness_stack_in)
```

Comment:
Using const& forces a copy in this class (always). Passing the object itself (like clang-formatter suggests) makes the caller manage it. If the called had a const& a copy will be made (so no changes), if the caller had an object already, it can pass ownership with std::move and then 0 copies are made.